lib/structuraid_core/engineering/locations/relative.rb

```diff
@@ -4,11 +4,39 @@ module Locations
       class Relative < Base
         attr_accessor :value_1, :value_2, :value_3
```

Comment:
Does this need to be `attr_accessor`?
```suggestion
        attr_reader :value_1, :value_2, :value_3
```

Reply:
Ok, done. But will add a method to update these values

Reply:
Never mind, it Is implemented, jaja. I just forgot to change the type of accessor to reader xD, done!